internal/reconstitution/cache.go

```diff
@@ -95,7 +95,7 @@ func (c *cache) Fill(ctx context.Context, comp *apiv1.Composition, synthesis *ap
 	compNSN := types.NamespacedName{Name: comp.Name, Namespace: comp.Namespace}
 	c.synthesesByComposition[compNSN] = append(c.synthesesByComposition[compNSN], synKey.Generation)
 
-	logger.Info("cache filled")
+	logger.V(0).Info("cache filled")
```

Comment:
I'm not too familiar with `go-logr` what are the "V" chained methods for?

Reply:
They just set the verbosity like the old glog/klog libraries did in the google tradition